blscosi/protocol/sub_protocol.go

```diff
@@ -218,7 +214,8 @@ func (p *SubBlsCosi) dispatchRoot() error {
 	case <-time.After(p.Timeout):
 		// It might be only the subleader then we send a notification
 		// to let the parent protocol take actions
-		log.Warn(p.ServerIdentity(), "timed out while waiting for subleader response")
+		log.Warn(p.ServerIdentity(),
+			"timed out while waiting for subleader response")
```

Comment:
I don't get why your editor is always changing the max line width but not consistently, there is a bunch of small and irrelevant such patches. Is there a way to avoid that?

Reply:
There still is a 80-character limit in the code-guidelines, but my editor only imposes it if I edit the line.

We could add this to the `pre-commit` list of things to do. But then I'll propose a 100-character limit...

Reply:
Haa, that's why! Yep, a `pre-commit` is the way to go.